websites/Y/YouTube Music/dist/metadata.json

```diff
@@ -16,7 +16,7 @@
     "zh_TW": "全新的音樂服務，包含適用於Android，iOS和PC的官方專輯，單曲，視頻，混音，現場表演等。都在這裡！"
   },
   "url": "music.youtube.com",
-  "version": "2.4.4",
+  "version": "2.4.7",
```

Comment:
The actual version is 2.4.6, So I increase it's to 2.4.7

By the way, this change should that I have increase from 2.4.4 to 2.4.7 that because my own fork that have worst brance management. Will fix after this pr. Sorry

Reply:
Yeah make sure to update it